src/pages/research_profile/ResearchProfile.tsx

```diff
@@ -280,11 +300,21 @@ const ResearchProfile = () => {
 
         <Divider sx={{ my: '1rem' }} />
 
-        <Typography id="project-label" variant="h2" sx={{ mt: '1rem' }}>
+        <Typography variant="h2" gutterBottom sx={{ mt: '1rem' }}>
           {projectHeading}
         </Typography>
+        <Typography>
```

Comment:
Må vurdere om denne alltid skal vises, eller bare vises når personen faktisk har resultater.
Rart om den forsvinner når bruker endrer paginering også.
Samme gjelder for prosjekter.

Reply:
Etter diskusjon fant vi ut at den helst burde vises bare når personen har treff. Som vanskeliggjør det enda mer siden alle treff forsvinner når man endrer paginering eller sortering 😩